spec/rails_helper.rb

```diff
@@ -81,14 +81,12 @@
 end
 
 Selenium::WebDriver.logger.level = :warn
-Capybara.register_driver :chrome do |app|
-  options = Selenium::WebDriver::Chrome::Options.new(args: %w[headless disable-gpu])
-  browser_path = ENV['DEVELOPMENT_SELENIUM_CHROMIUM_BROWSER_PATH']
-  options.binary = browser_path if browser_path.present?
-  Capybara::Selenium::Driver.new app, browser: :chrome, options: options
+Capybara.register_driver :firefox do |app|
```

Comment:
Bon, évidemment le changement de driver, ça plante pour moi. C'est supposé marcher out of the box ou tu as dû faire qqch ?

Je te cache pas que faire en sorte que ça marche avec chrome a déjà été un peu tricky pour moi, et j'ai un peu peur des conséquences d'un changement (d'autant que firefox est mon navigateur par défaut de tous les jours). 

Je veux bien que tu m'en dises + sur la manip, et si ça paraît trop hasardeux, en faire un truc configurable avec des variables d'environnement et laisser chrome par défaut ?

Reply:
J'ai rien fait d'autre que les changements de ce commit.
Le mode headless ne fonctionne pas et ça m'énerve un peu d'avoir une fenêtre qui s'ouvre à chaque test, on se croirait en java.
Je veux bien le supprimer, j'espère juste que je vais arriver à lancer les tests avec chrome